notify/notify.go

```diff
@@ -407,27 +440,5 @@ func (n *InhibitNotifier) Notify(ctx context.Context, alerts ...*types.Alert) er
 		}
 	}
 
-	return n.notifier.Notify(ctx, filtered...)
-}
-
-// LogNotifier logs the alerts to be notified about. It forwards to another Notifier
-// afterwards, if any is provided.
-type LogNotifier struct {
```

Comment:
Yes no longer used. Just noticed myself while reviewing that I dropped it 😁 . I'll investigate where it makes most sense to add some debug logging.